howdoi/howdoi.py

```diff
@@ -372,6 +379,29 @@ def _clear_cache():
     return cache.clear()
 
 
+def _parse_json(res, args):
+    """
+    @res: json object with answers and metadata
+    @args: command-line arguments (used for parsing)
+    returns: formated string of text ready to be printed
+    """
+    res = json.loads(res)
+    if "error" in res:
+        return res["error"]
+
+    splitter_length = 80
+    answer_splitter = '\n' + '=' * splitter_length + '\n\n'
```

Comment:
In PR #280 @tayoogunbiyi added a `build_splitter` function which should be nice to use here

Reply:
Looks like PR #280 hasn't been merged to master yet, so we're currently waiting to make this change.

Reply:
#280 was merged, so you can pull in those functions now (and also fix a few merge conflicts)